unity-client/Assets/StreamingAssets/PerformanceTestRunInfo.json

```diff
@@ -51,7 +51,7 @@
         "ScriptingRuntimeVersion": ""
     },
     "TestSuite": "",
-    "StartTime": 1605525670275.5386,
+    "StartTime": 1606218909973.981,
```

Comment:
This is harmless, but let's try to avoid commiting unrelated files

Reply:
I assume that this file should go into the .gitignore?

Reply:
That's actually a good question. Maybe it should, @BrianAmadori what do you think about it?